docs/source/guide/guide.myst

```diff
@@ -0,0 +1,17 @@
+# Users Guide
+
+# Welcome to mitiq's documentation!
```

Comment:
```suggestion
Welcome to mitiq's documentation!
```

Otherwise a _Welcome to mitiq's documentation!_ appears in the main TOC of the docs.

Reply:
Yes, there is this TOC issue. Actually, I think it's fine to have the welcome only in the index, so we can remove the line completely here.

Reply:
I have removed it, for some reason I had 2 first level headings in that file, it should just be the `#User Guide`